config/betterquesting/Readme.md

```diff
@@ -49,7 +49,7 @@ https://try.github.io/
 3. Before you begin making changes, you want to click `Fetch origin` to get the latest updates and then start a new branch (`Branch`, then `New Branch`). If you followed 2. that will automatically be based on the GTNewHorizons/GT-New-Horizons-Modpack master branch.
 4. Start GTNH. You need the full game, not a development environment. You should also check that you are using the latest BQ from https://github.com/GTNewHorizons/BetterQuesting/releases (you should only use normal versions, not pre-versions).
 5. Replace the DefaultQuests folder in your config/betterquesting game files with the one from your github folder. That means to delete the existing folder and then copy in the one from github, do not merge them!
-6. Then do `/bq_admin default load` in the game.
+6. Run `/bq_admin default load` in the game chat. If the command crashes your client, wait a bit before running it. If it crashes again, wait for up to a minute.
```

Comment:
i never seen it crash . maybe it wont work when the loading progress is active

Reply:
It crashed today for me. [Discord link](https://discord.com/channels/181078474394566657/949447391587733504/1104033775505121280). The waiting for around 30 seconds helped. Was working with singleplayer GTNH 2.3.3.

If the discord link doesn't load, it was today in #quest-dev.

Reply:
can you link it ?

Reply:
As the link above, https://discord.com/channels/181078474394566657/949447391587733504/1104033775505121280

Or did you mean something different?

Reply:
nope all fine.